svg.js.d.ts

```diff
@@ -1,3 +1,8 @@
+export = svgjs;
+export as namespace svgjs;
+
+declare var svgjs: svgjs.Library;
+

```

Comment:
Great. 🍻  

> we would just have to fix the default export and the namespace error.

Like you already said in the comment bellow. I will try to do some work on this issue.